prow/release-commit.sh

```diff
@@ -83,6 +83,7 @@ dashboards:
   istio-service-dashboard: 7636
   istio-workload-dashboard: 7630
   pilot-dashboard: 7645
+  istio-extension-dashboard: 7646
```

Comment:
@douglas-reid @howardjohn do you know how does this number being set? I just choose one after 7645.

Reply:
You need to actually create it in grafana.com and it will get auto assigned. I can do it if you do not have permissions

Reply:
Offline discussed, after this PR gets merged in, we will replace this ID with a real one.